tests/generated.go

```diff
@@ -163,14 +163,8 @@ type pairServiceNew struct {
 	HasCredential bool
 	Credential    string
 	// Optional pairs
-	HasDefaultContentType  bool
```

Comment:
Why they are been removed?

Reply:
There's no operation in `service` namespace has the `content_type` pair. So I modified the way to get the defaultable pairs in https://github.com/beyondstorage/go-storage/pull/821/files#diff-0ca1d3b96d3cc7d55c46d876668a196e55b5a73e0208c01307c5d93e93d0bc9cR311-R312

Reply:
Makes sense.